Missing install step in module makefile

Hi there,
I am trying to build this library from source to work on our 32 bits pc104, since the version provided by [icus-os-files](https://github.com/icub-tech-iit/icub-os-files) appears to be 64bits (see https://github.com/robotology/icub-tech-support/issues/1106  for why I am trying to do this).

During the install step, this repo's main `Makefile` tries to install the `module` using `cd module && $(MAKE) install`, but [the `module` folder's Makefile](https://github.com/robotology/cfw002/blob/master/src/module/Makefile) does not have an install step, which creates the following error:

```bash
cd module && make install
make[1]: Entering directory '/usr/local/src/robot/cfw002/src/module'
make[1]: *** No rule to make target 'install'.  Stop.
make[1]: Leaving directory '/usr/local/src/robot/cfw002/src/module'
make: *** [Makefile:16: install] Error 2
```

The `install` step existed in [the previous version of the file](https://github.com/robotology/cfw002/blob/c8be0d3a49b4db9d8406c662325a145ef7bd407a/src/module/Makefile):

```bash

KVER=$(shell uname -r)
KSRC=/lib/modules/$(KVER)/build
MODULE_INSTALL_PATH=/lib/modules/$(KVER)/iCubDrivers/cfw002/LinuxDriver
MODULE_TARGET=cfw002.ko

obj-m := cfw002.o

###
...
###

install: $(MODULE_TARGET)
	mkdir -p $(MODULE_INSTALL_PATH)
	cp $(MODULE_TARGET) $(MODULE_INSTALL_PATH)/
```

Should it be reincluded in the file? Or should the `cd module && $(MAKE) install` be removed from the main repo's Makefile?

--
EDIT: I re-added the missing lines in the file based on the previous version, and so far it seems to work. I can create a PR if someone confirms that it is not hardware-specific.

--- a/src/module/Makefile
+++ b/src/module/Makefile
@@ -1,6 +1,8 @@
 #VER=2.6.29.2
 KVER=$(shell uname -r)
 KSRC=/lib/modules/$(KVER)/build
+MODULE_INSTALL_PATH=/lib/modules/$(KVER)/iCubDrivers/cfw002/LinuxDriver
+MODULE_TARGET=cfw002.ko
 
 obj-m := cfw002.o
 
@@ -16,3 +18,7 @@ driver:
 clean: 
 	$(MAKE) -C $(KSRC) M=$(PWD) clean
 	@rm -f Module.symvers
+
+install: $(MODULE_TARGET)
+	mkdir -p $(MODULE_INSTALL_PATH)
+	cp $(MODULE_TARGET) $(MODULE_INSTALL_PATH)/

